packages/victory-selection-container/src/victory-selection-container.tsx

```diff
@@ -120,4 +120,3 @@ export function selectionContainerMixin<
 
 export const VictorySelectionContainer =
   selectionContainerMixin(VictoryContainer);
-export type VictorySelectionContainer = typeof VictorySelectionContainer;
```

Comment:
These types are no longer necessary for extending types in the `victory-native` package